x-pack/plugins/stack_alerts/public/alert_types/geo_containment/query_builder/util_components/geo_index_pattern_select.tsx

```diff
@@ -87,18 +94,14 @@ export class GeoIndexPatternSelect extends Component<Props, State> {
             >
               <FormattedMessage
                 id="xpack.stackAlerts.geoContainment.noIndexPattern.doThisLinkTextDescription"
-                defaultMessage="create an index pattern"
+                defaultMessage="create an index pattern."
```

Comment:
Same as above, maybe capitalize `create` since we're in here and it's directly lifted off the other one